dandi/dandiapi.py

```diff
@@ -295,6 +296,16 @@ def create_dandiset(self, name: str, metadata: Dict[str, Any]) -> "RemoteDandise
             self, self.post("/dandisets/", json={"name": name, "metadata": metadata})
         )
 
+    def match_schema_version(self, schema_version: Optional[str] = None) -> None:
```

Comment:
I am hopeful in a bright(er) future, could we make it a

```suggestion
    def check_schema_version(self, schema_version: Optional[str] = None) -> None:
```
?

Reply:
Are you asking for any behavior change or just a name change?

Reply:
just the name.  ATM it is equality (AKA "match") check but later could become more sophisticated, and thus better make the name more descriptive at once.
Later, we might need to adjust it anyways: filed https://github.com/dandi/dandi-api/issues/437

Reply:
Done.